src/nstime.c

```diff
@@ -152,12 +152,41 @@ nstime_monotonic_impl(void) {
 }
 nstime_monotonic_t *JET_MUTABLE nstime_monotonic = nstime_monotonic_impl;
 
+static void
+nstime_get_realtime(nstime_t *time) {
+  struct timespec ts;
+
+  clock_gettime(CLOCK_REALTIME, &ts);
+  nstime_init2(time, ts.tv_sec, ts.tv_nsec);
+}
+
+static bool
+nstime_prof_update_impl(nstime_t *time) {
+  nstime_t old_time;
+
+	nstime_copy(&old_time, time);
+
+  if (unlikely(opt_prof_time_resolution == prof_time_resolution_high)) {
```

Comment:
Is this unlikely? I suspect we'll turn it on by default.

Reply:
OK - I'll drop the unlikely().